rudder_airflow_provider/version.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.1'
\ No newline at end of file
+__version__ = '1.0.2'
```

Comment:
Ideally this we should make 1.1.0 if we are upgrading airflow version required to higher.